extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -397,6 +397,8 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
  */
 function handleDismiss(state) {
   hideBanner(state);
+  // Restore focus to document after dismiss for screen readers.
+  state.element.ownerDocument.body./*REVIEW*/focus();
```

Comment:
If we find that this problem is not related to fixedLayer, I suggest we do what I described above in `removeElement` function instead of `removeFromFixedLayer`